.github/workflows/deploy.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy to TestFlight
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Number version to deploy'
+        required: true
+
+jobs:
+  bump_version:
+    name: Deploy 
+    runs-on: macos-10.15
+    defaults:
+      run:
+        working-directory: BuyPolish
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 
+          token: ${{ secrets.WORKFLOW_ACCESS_TOKEN_GITHUB }}
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+      - name: Cache bundler gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Cache pods
+        uses: actions/cache@v2
+        with:
+          path: BuyPolish/Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
+      - name: Bundler
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - name: Pod install
+        run: bundle exec pod install
+      - name: Update Firebase config
+        run: |
+          rm Pola/GoogleService-Info.plist
+          echo $DEV_ENV_FILE >> Pola/GoogleService-Info.plist
+          cat Pola/GoogleService-Info.plist
```

Comment:
Are you sure about it?

Reply:
We need a replace firebase configuration file before deployment. As I understood and check the content of this file shouldn't be visible on the console. If you know a better action/way to do this tell me and I will change it :)

Reply:
I'm just wondering if we really need to print this value to console.  I understand that Github does everything it can to mask this data, but if this feature fails for some reason (e.g. insert a new line, bit flip), the secret will be visible.  If we want to check if the file has been created, we can count the number of characters.
```
cat Pola/GoogleService-Info.plist | wc -c
```

Reply:
Maybe I'm wrong, but `echo $DEV_ENV_FILE >> Pola/GoogleService-Info.plist` doesn't print to console, because the output is redirected to file. So even if something goes wrong, the secret will be only in this file.

Reply:
On line 52, the secret is saved to file.
On. line 53, the secret is printed on the console.